ci_build_images/msan.motd

```diff
@@ -0,0 +1,60 @@
+This is a container for ASAN, UBSAN and MSAN building.
+
+It can also be used as a basic build/test that contains rr.
+
+rr is available if the container is started with --privileged.
+
+The following assumed that /source is a mounted volume of the
+MariaDB source code.
+
+A basic MSAN build can be configured with:
+
+  cmake \
+        -DWITH_MSAN=ON  \
+        -DCMAKE_{EXE,MODULE}_LINKER_FLAGS="-L${MSAN_LIBDIR} -Wl,-rpath=${MSAN_LIBDIR}" \
+        -DWITH_INNODB_{BZIP2,LZ4,LZMA,LZO,SNAPPY}=OFF \
+        -DCMAKE_DISABLE_FIND_PACKAGE_URING=1 \
```

Comment:
These instructions are specific to this container, right? If the `-dev` packages of the various compression libraries are missing from the container, then there should be no need to disable them, right?

Furthermore, thanks to MariaDB/server#4257, there should be no reason to disable the use of `liburing-dev` (which I think you wrote would be included in the container image).